src/Log.ml

```diff
@@ -24,7 +24,9 @@ let indented_sep (indent : int) = "\n" ^ (String.make (42 + indent) ' ')
 
   let is_enabled = ref false
   let do_log ~tags level lstr =
-    if !is_enabled then Log.string !logger ~tags ~level (Lazy.force lstr)
+    let open Core
+     in prerr_endline (Lazy.force lstr)
+    (* if !is_enabled then Log.string !logger ~tags ~level (Lazy.force lstr) *)
```

Comment:
Please rebase with `master` branch and you wouldn't need this with the latest `Log.ml`.